src/Tasks/SignFile.cs

```diff
@@ -36,6 +39,9 @@ public SignFile()
 
         public string TimestampUrl { get; set; }
 
+#if RUNTIME_TYPE_NETCORE
+        [SupportedOSPlatform("windows")]
```

Comment:
I don't think this should propagate quite so far. If we're going to expose the task we should expose it everywhere and emit good error messages if we hit a non-windows-compat codepath.

Reply:
I removed it from the Execute and appreciate this new way better than how it was. I'm not sure if we should go in and log errors in other areas where I applied `SupportedOSPlatform`? I think it makes sense to keep the static SignFile methods with this attribute, at least.